src/AppBundle/Entity/Post.php

```diff
@@ -95,7 +95,7 @@ class Post
      *      mappedBy="post",
      *      orphanRemoval=true
      * )
-     * @ORM\OrderBy({"publishedAt" = "DESC"})
+     * @ORM\OrderBy({"publishedAt": "DESC"})
```

Comment:
@stof You are totally right! I think I didn't finished my thoughts :D. After thinking 3 times about the fact I think `:` is much cleaner. 
I never thought about that the shortcut syntax misses `value=` actually.

And yeah, I meant the FrameworkExtraBundle, not the FrameworkBundle.